BO4E/BO/Bilanzierung.cs

```diff
@@ -0,0 +1,137 @@
+﻿using BO4E.COM;
+using BO4E.ENUM;
+using BO4E.meta;
+
+using Newtonsoft.Json;
+
+using ProtoBuf;
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace BO4E.BO
+{
+    /// <summary>
+    /// Bilanzierungs BO
+    /// </summary>
+    [ProtoContract]
+    public class Bilanzierung : BusinessObject
+    {
+        /// <summary>
+        /// Eine Liste der verwendeten Lastprofile (SLP, SLP/TLP, ALP etc.)
+        /// </summary>
+        [JsonProperty(PropertyName = "lastprofile", Required = Required.Default)]
+        [JsonPropertyName("lastprofile")]
+        [NonOfficial(NonOfficialCategory.CUSTOMER_REQUIREMENTS)]
+        [ProtoMember(1001)]
+        public List<Lastprofil> Lastprofile { get; set; }
+        /// <summary>
+        /// Start der Bilanzierung
+        /// </summary>
+        [JsonProperty(PropertyName = "bilanzierungsbeginn", Required = Required.Default)]
+        [JsonPropertyName("bilanzierungsbeginn")]
+        [NonOfficial(NonOfficialCategory.CUSTOMER_REQUIREMENTS)]
+        [ProtoMember(1002)]
+        public DateTimeOffset Bilanzierungsbeginn { get; set; }
+        /// <summary>
+        /// Ende der Bilanzierung
+        /// </summary>
+        [JsonProperty(PropertyName = "bilanzierungsende", Required = Required.Default)]
+        [JsonPropertyName("bilanzierungsende")]
+        [NonOfficial(NonOfficialCategory.CUSTOMER_REQUIREMENTS)]
+        [ProtoMember(1003)]
+        public DateTimeOffset Bilanzierungsende { get; set; }
+
+        /// <summary>
+        /// Bilanzkreis
+        /// </summary>
+        [JsonProperty(PropertyName = "bilanzkreis", Required = Required.Default)]
+        [JsonPropertyName("bilanzkreis")]
+        [NonOfficial(NonOfficialCategory.CUSTOMER_REQUIREMENTS)]
+        [ProtoMember(1004)]
+        public string Bilanzkreis { get; set; }
+
+        /// <summary>
+        ///     jahresverbrauchsprognose
+        /// </summary>
+        [JsonProperty(PropertyName = "jahresverbrauchsprognose", Required = Required.Default)]
+        [JsonPropertyName("jahresverbrauchsprognose")]
+        [NonOfficial(NonOfficialCategory.CUSTOMER_REQUIREMENTS)]
+        [ProtoMember(1005)]
+        public Menge Jahresverbrauchsprognose { get; set; }
```

Comment:
wäre "Verbrauch" nicht das bessere COM?

Reply:
Verbrauch ist halt immer Zeitscheibenbezogen (mit Start und Ende). Das haben wir in dem Fall aber ja gar nicht gegeben